image.yaml

```diff
@@ -213,8 +213,8 @@ packages:
     - openssl
 artifacts:
   - name: jboss-datagrid-server.zip
-    description: "Artifact (patch) is available on Customer Portal: https://access.redhat.com/jbossnetwork/restricted/softwareDownload.html?softwareId=81521"
-    md5: bd4d8516223e5257ae7fe8a00feb1b3b
+    description: "Pre-release artifact (CR1), not available in customer portal"
```

Comment:
The released artifact (patch) is available on Customer Portal at https://access.redhat.com/jbossnetwork/restricted/softwareDownload.html?softwareId=8762, so maybe it would be better to point at it in the description instead of stating it's not available.

Reply:
Updated according to the suggestion